wurst/systems/units/hostiles/AncientHydra.wurst

```diff
@@ -73,6 +73,8 @@ public class AncientHydra extends Hostile
         for i = 0 to 1
             new LesserHydra(pos)
 
+        super.onDeath()
```

Comment:
So was this supposed to be here? Quazz at first said no but now its still here

Reply:
@Marsunpaisti I actually think it is supposed to be there now